yarn.lock

```diff
@@ -474,6 +486,11 @@
     "@nodelib/fs.scandir" "2.1.5"
     fastq "^1.6.0"
 
+"@pkgjs/parseargs@^0.11.0":
```

Comment:
info Reasons this module exists
   - "tsup#sucrase#glob#jackspeak" depends on it
   - Hoisted from "tsup#sucrase#glob#jackspeak#@pkgjs#parseargs"